packages/newspringchurchapp/src/tabs/connect/Connect.js

```diff
@@ -15,6 +15,7 @@ import {
   Icon,
   PaddedView,
 } from '@apollosproject/ui-kit';
+import UserPrayers from '../../ui/UserPrayer';
```

Comment:
I believe the UserPrayerCard is a great name, however, this component represents the entire view of user prayers so I don't think just UserPrayer is descriptive enough. I'd recommend something like UserPrayerList.